app/views/course/users/invitations.html.slim

```diff
@@ -6,7 +6,8 @@
 - accepted_course_users = @course_users.each.select { |user| !user.invited? }.size
 - progress = accepted_course_users * 100 / [@course_users.length, 1].max
 = display_progress_bar progress, [] do
-  = t('.progress', accepted: accepted_course_users, total: @course_users.size)
+  = t('course.users.invitations.progress', accepted: accepted_course_users,
```

Comment:
Also did you see that the invitations page' exp progress bar follows the size of the invitations bar?